_books/ion-1-1/src/modules/system_module.md

```diff
@@ -39,18 +51,19 @@ The Ion 1.1 System Symbol table _replaces_ rather than extends the Ion 1.0 Syste
 | 14 | `macro_table`                   |
 | 15 | `symbol_table`                  |
 | 16 | `module`                        |
-| 17 | `retain`                        |
-| 18 | `export`                        |
-| 19 | `catalog_key`                   |
+| 17 | ~~retain~~                      |
+| 18 | export                          |
+| 19 | ~~catalog_key~~                 |
 | 20 | `import`                        |
-| 21 | _&lt;empty string>_ (i.e. `''`) |
+| 21 | _zero-length text_ (i.e. `''`)  |
```

Comment:
Remind me when/where this gets used? It's two bytes to refer to the empty symbol vs `0xA0` to refer to a symbol with inline empty text.

Reply:
It's needed for the FlexSym encoding of `''`. We can't use `A0`/`90` for empty & unknown symbol in FlexSyms anymore because they're in the range that we've claimed for system symbols. Now we use `0` in the system symbol range for unknown text, and this symbol for the empty string symbol text. Of course, we can revisit the exact ID of this symbol if it would be helpful to do so.